.github/workflows/load-test.yml

```
@@ -1,13 +1,112 @@
-name: Test Stub
+name: Load test
+
 on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
   workflow_dispatch:
+    inputs:
+      test-id:
+        type: string
+        description: "Identifier label for Datadog metrics"
+        default: "server-load-test"
+      k6-test-path:
+        type: string
+        description: "Path to load test files"
+        default: "perf/load/*.js"
+      k6-flags:
+        type: string
+        description: "Additional k6 flags"
+      api-env-url:
+        type: string
+        description: "Override API base URL"
+        default: "https://api.qa.bitwarden.pw"
```

```suggestion
        description: "URL of the API environment"
        default: "https://api.qa.bitwarden.pw"
      identity-env-url:
        type: string
        description: "URL of the Identity environment"
        default: "https://identity.qa.bitwarden.pw"
```
⛏️ This is what's in the relay file. It's not an override if it has a default.